### ISSUE TITLE
ERROR: 'getHtmlContent' is not defined

### ISSUE THREAD

**Issue by RahulSDeshpande** [daothanhduy305/KRichEditor#15]
Sample:
```
var updatedHtmlReport: String? = null
kRichEditorFragment.editor.getHtmlContent { html -> updatedHtmlReport = html }
```

When I try to execute the above code, I get this in the logcat!
```
[INFO:CONSOLE(1)] "Uncaught ReferenceError: getHtmlContent is not defined",
                   source: file:///android_asset/richEditor.html
```

`updatedHtmlReport` doesn't get updated.

Cheers!!

**Comment by RahulSDeshpande** [comment]
I even checked your sample app.
In that too the `getHtmlContent()` doesn't work properly :(

In `RichEditor.kt`:
```
private fun getHtmlContent(callBack: ValueCallback<String>) = load("javascript:getHtmlContent()", callBack)
```

There is some issue with the code below inside `kricheditor.js`?:
```
function getHtml() { return quill.root.innerHTML; }
```

There is no such method like `getHtmlContent()` in `kricheditor.js`.
It should be `getHtml()`.

Please validate & correct me.

**Comment by RahulSDeshpande** [comment]
Any update @daothanhduy305?

**Comment by daothanhduy305** [comment]
> Any update @daothanhduy305?

Okay, I see the issue. I accidentally had all the "getHtml" be replaced by "getHtmlContent" during the refactoring. Will fix asap. Thank you for your report :p.

**Comment by RahulSDeshpande** [comment]
Btw I tried fixing this issue, it didn't work. :(

Check the changes I did in my forked repo:
https://github.com/RahulSDeshpande/KRichEditor/commit/cb2815747704b3d77b6da766be696bf0224a9f23

**Comment by RahulSDeshpande** [comment]
@daothanhduy305

Please check this PR:
https://github.com/daothanhduy305/KRichEditor/pull/16

Have fixed the issue.
Let me know if its good to merge.

Cheers :)

**Comment by daothanhduy305** [comment]
Merged and fixed at cb2815747704b3d77b6da766be696bf0224a9f23
